client/my-sites/sharing/connections/services/eventbrite.js

```diff
@@ -1,173 +1,59 @@
 /**
  * External dependencies
  */
-import React, { Component, PropTypes } from 'react';
-import classnames from 'classnames';
-import { connect } from 'react-redux';
-import { identity, last, replace, some } from 'lodash';
-import { localize } from 'i18n-calypso';
-import SocialLogo from 'social-logos';
+import { PropTypes } from 'react';
+import { last, some } from 'lodash';
 
 /**
  * Internal dependencies
  */
-import Connection from '../connection';
+import { SharingService, connectFor } from '../service';
```

Comment:
This should be a full path relative to `client`, ie. `my-sites/sharing/connections/services`.